### PR TITLE
Update BitcoinTicker widget to handle other cryptocurrencies 

### LINKED ISSUE #2470: Bitcoin Ticker keeps showing locale currency sign
My locale is GB, so if I don't specify a currency, it will show BTC in £. But when I specify, say,"USD", it shows the USD **numbers** but still shows the "£" sign in front of it.

So let's say BTC is "$44,000" in USD and "£31,000" in GBP. If I don't specify, it will say "£31,000". But if I write "USD", it says "**£**44,000" (the USD value, but the GBP sign).

### PR DESCRIPTION
**Pull request by Graeme22**
Very simple changes to allow the BitcoinTicker widget to become more generic and support a host of other cryptocurrencies.

Edit: Also fixes #2470 by allowing users to input their currency symbol instead of trying to guess it.

This *clearly* needs to be squashed.

### CODE REVIEW

File: libqtile/widget/crypto_ticker.py
Comment (ramnes) [inline libqtile/widget/crypto_ticker.py:55]: Let's make the migration simpler by not renaming tihs. :)

File: libqtile/widget/crypto_ticker.py
Comment (ramnes) [inline libqtile/widget/crypto_ticker.py:82]: I'd rather not rename this one either.

**Review by ramnes — CHANGES_REQUESTED** [review]
Needs a [migration](https://github.com/qtile/qtile/blob/master/libqtile/scripts/migrate.py) if anything. :)

Tests would also be welcome, but if there aren't any already for this widget let's say it's up to your goodwill.

**Comment by ramnes** [comment]
Also, I think you should rebase on master and squash the commits.

**Comment by elParaguayo** [comment]
> Needs a [migration](https://github.com/qtile/qtile/blob/master/libqtile/scripts/migrate.py) if anything. :)
> 
> Tests would also be welcome, but if there aren't any already for this widget let's say it's up to your goodwill.

In the issue thread, I had volunteered to make the changes to the widget and write a test. I did the first part but didn't push it as I hadn't finished the test. I then had a very busy week last week.

I'm still happy to write a test for this one but will wait until you've merged the widget.

**Comment by Graeme22** [comment]
For the test, is it ok to include the API call to coinbase, or do we want it to be deterministic?

**Comment by elParaguayo** [comment]
We'd need to mock the API call so that we get predefined data back.

**Comment by elParaguayo** [comment]
@Graeme22 I don't profess to be an expert at writing tests but I've written a lot of the widget tests here. Have a look in the test folder to see how you can mock different objects. 

Happy to help where I can (including writing the test if you don't want to).

**Comment by Graeme22** [comment]
> @Graeme22 I don't profess to be an expert at writing tests but I've written a lot of the widget tests here. Have a look in the test folder to see how you can mock different objects. 
> 
> Happy to help where I can (including writing the test if you don't want to).

I'll give it a shot, I like being able to contribute at least a little to such a great project :)

**Comment by Graeme22** [comment]
I messed up my fork somehow, let me reopen a new PR.

**Pull request closed without merging**
